etna/transforms/missing_values/imputation.py

```diff
@@ -70,7 +112,9 @@ def __init__(
 
             - If "forward_fill" then replace missing dates using last existing value
 
-            - If "seasonal" then replace missing dates using seasonal moving average
+            - If "seasonal" then replace missing dates using seasonal moving average in autoregressive manner
+
+            - If "seasonal_statistics" then replace missing dates using seasonal moving average on existing values
```

Comment:
Could you explain why do you suggest this name?

Reply:
I couldn't come up with better naming, do you have any ideas?

Reply:
If it isn't autorregressive how does it work with missing values? It isn't clear from the docs. We should clarify the differences.